src/main/java/at/spengergasse/cooking/recipes/domain/Recipe.java

```diff
@@ -0,0 +1,15 @@
+package at.spengergasse.cooking.recipes.domain;
+
+import lombok.AllArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document("recipes")
+@AllArgsConstructor
+public class Recipe {
+    @Id
+    private String key;
+    public String username;
```

Comment:
why not modeled as UserRef object?

Reply:
https://github.com/Die-Spengergasse/sj2324-5ehif-classproject-recipes/pull/8